Android/FirebaseJobDispatcher/source/Firebase.JobDispatcher/Properties/AssemblyInfo.cs

```diff
@@ -9,5 +9,5 @@
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 
-[assembly: AssemblyVersion("0.6.0.0")]
-[assembly: AssemblyFileVersion("0.6.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("0.7.0.0")]
```

Comment:
We do not  rev AssemblyVersion.  For bindings, the assembly version is usually just 1.0.0.0, and then we set the file version to the library version.  I suggest picking a version number and sticking with it. AssemblyFileVersion you can change.